arrange.py

```diff
@@ -109,26 +99,14 @@ def is_there_edge_between_players(angel_player, mortal_player):
         players_are_from_same_house = get_house_from_player(
             angel_player) == get_house_from_player(mortal_player)
 
-
-    # Relax no same CG requirement
-    random_relax_cg_requirement = random.random() < RELAX_NO_SAME_CG_REQUIREMENT_PERCENTAGE
-    if random_relax_cg_requirement:
-        players_are_from_same_cg = False
-    else:
-        players_are_from_same_cg = get_cg_from_player(
-            angel_player) == get_cg_from_player(mortal_player)
-
-
-    valid_pairing = gender_pref_is_respected and (not players_are_from_same_house) and (not players_are_from_same_cg)#and (not players_are_from_same_faculty) # Remove same-house reqr -->  #or players_are_from_same_house) and
+    valid_pairing = gender_pref_is_respected and (not players_are_from_same_house) #and (not players_are_from_same_faculty) # Remove same-house reqr -->  #or players_are_from_same_house) and
```

Comment:
Delete unused code please. Don't leave it as a comment. You may use comments when experimenting (when you're unsure of what you might need), but it's not good practise to be kept in production code